image.go

```diff
@@ -554,30 +554,30 @@ func (i *Image) DrawTriangles(vertices []Vertex, indices []uint16, img *Image, o
 	vs := i.ensureTmpVertices(len(vertices) * graphics.VertexFloatCount)
 	dst := i
 	if options.ColorScaleMode == ColorScaleModeStraightAlpha {
-		for i, v := range vertices {
-			dx, dy := dst.adjustPositionF32(v.DstX, v.DstY)
+		for i := range vertices {
```

Comment:
Let's add a comment to explain why we don't use the second variable at this for-range loop, like

```go
// Avoid the value variable at this for-range loop since this would cause unnecessary copyings (#3103).
```

(Please refine this sentence :-)

Reply:
OK, PR updated and force-pushed; the comment has been added.